# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pixieplus/shared-files.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -urN pixieplus.orig/kdenonbeta/pixieplus/mimetypes/Makefile.am pixieplus/kdenonbeta/pixieplus/mimetypes/Makefile.am
--- pixieplus.orig/kdenonbeta/pixieplus/mimetypes/Makefile.am	2005-03-29 14:47:27.437200680 +0200
+++ pixieplus/kdenonbeta/pixieplus/mimetypes/Makefile.am	2005-03-29 15:51:20.607470224 +0200
@@ -1,5 +1,5 @@
 
-mime_DATA = x-pcx.desktop x-xwd.desktop x-miff.desktop x-tga.desktop \
+mime_DATA = x-xwd.desktop x-miff.desktop x-tga.desktop \
 	x-pict.desktop
 
 mimedir = $(kde_mimedir)/image
